viz.yaml

```diff
@@ -441,7 +435,7 @@ publish:
       wu_data_15_sum_json: wu_data_15_sum_json
       wu_state_data_json: wu_state_data_json
     context:
-      resources: ["lib_d3_js", "topojson", "all_css", "all_js_minified"]
+      resources: ["all_css", "all_js_minified"]
```

Comment:
not sure if this matters, but we now have css in front of js for the resources. Is that intentional?

Reply:
oops, not intentional and I think it would be smart to switch them.